lib/sinon/call.js

```diff
@@ -155,7 +155,7 @@
             },
 
             toString: function () {
-                var callStr = this.proxy.toString() + "(";
+                var callStr = this.proxy ? this.proxy.toString() + "(" : "";
                 var args = [];
 
                 for (var i = 0, l = this.args.length; i < l; ++i) {
```

Comment:
You forgot [the other part](https://github.com/cjohansen/Sinon.JS/issues/648#issuecomment-118257844) of the hotfix:
```javascript
 if (!this.args){
   return ":(";
 }
```
Without it this breaks with 
```
	TypeError: 'undefined' is not an object (evaluating 'this.args.length')
	    at /Users/carl-erik.kopseng/dev-priv/Sinon.JS/pkg/sinon.js:2338
        ...
```
([ref](https://github.com/cjohansen/Sinon.JS/issues/648#issuecomment-139385678))